src/download/index.test.ts

```diff
@@ -115,108 +104,56 @@ describe("Download Command", () => {
   });
 
   setup()
-    .do(() => DownloadCommand.run(["--group-by", "CC API Family"]))
+    .do(() => DownloadCommand.run([]))
     .it("downloads APIs", () => {
       expect("apis").to.be.a.directory();
-      expect("apis/api-config.json")
+      expect("apis/shopper-customers/.metadata.json")
         .to.be.a.file()
-        .with.content(`${API_CONFIG_JSON}\n`);
+        .with.content(API_CONFIG_JSON + "\n");
       expect("apis/shopper-customers").to.be.a.directory();
       expect("apis/shopper-customers/file.raml")
         .to.be.a.file()
         .with.content("zip content");
     });
 
   setup({ search: "test" })
-    .do(() =>
-      DownloadCommand.run(["--search=test", "--group-by", "CC API Family"])
-    )
+    .do(() => DownloadCommand.run(["--search=test"]))
```

Comment:
Am I misreading this or is there no validation on these tests other, possibly, the exit code?

Reply:
It's a bit hidden in the setup function. All --search does is change the URL for the exchange search, so there is a nock to verify that the correct URL is called. Because the response sent by nock is entirely under our control, there is no need to test the rest of it (doing so would just be duplicating other tests).